### PR TITLE
Feat: sell and footer sections

### PR DESCRIPTION
**Pull request by aspencer116**
Styling of the "sell" and footer sections plus change to the logo in "hero"

### CODE REVIEW

File: src/partials/_head.hbs [hunk 1]
```diff
@@ -5,6 +5,3 @@
 
 
 <link rel="stylesheet" href="/css/styles.css">
-
-<script src="https://code.jquery.com/jquery-1.10.2.js"></script>
-<script src="index.js"></script>
```
Comment (iAmNathanJ) [inline src/partials/_head.hbs:10]: I see you moved these to the footer. Good call.

Comment (aspencer116) [inline src/partials/_head.hbs:10]: Saw the comment on Rachel's and followed suit :)

File: src/partials/_sell.hbs [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+  <p class="sell--type">Have general quesitons about the A Modern Eden products, feel free to drop us a line.</p>
   <form class="sell--form" action="index.html" method="post">
-    <label for="name">name</label>
-    <input type="text" name="name" value="" placeholder="Your name">
-    <label for="email">email</label>
-    <input type="email" name="email" value="" placeholder="Your email">
-    <input type="textarea" name="message" placeholder="I would like to learn more about how my business can sell A Modern Eden products.">
-    <input type="submit" value="submit">
+    <label class="sell--label" for="name">name</label>
+    <input class="sell--input" type="text" name="name" value="" placeholder="Your name">
```
Comment (iAmNathanJ) [inline src/partials/_sell.hbs:7]: - I think the `for` attributes on these labels should refer to the `id`s of their respective inputs, but the inputs don't have `id`s. 
- If the `value` attr is empty, why not remove it?

File: src/partials/_sell.hbs [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+    <label class="sell--label__alt" for="email">email</label>
+    <input class="sell--input__alt" type="email" name="email" value="" placeholder="Your email">
+    <textarea class="sell--input__text" name="message" rows="5" placeholder="I would like to learn more about how my business can sell A Modern Eden products."></textarea>
+    <button type="submit" value="submit" class="sell--submit">submit</button>
```
Comment (iAmNathanJ) [inline src/partials/_sell.hbs:11]: I like your changes here (fix `textarea` and prefer using `buttons`)

**Pull request merged**